tuned.spec

```diff
@@ -329,9 +329,9 @@ if [ -r "%{_sysconfdir}/default/grub" ]; then
     %{_sysconfdir}/default/grub
 fi
 
-# move all user-defined profiles from /etc/tuned/ to /etc/tuned/profiles/
+# migrate all user-defined profiles from /etc/tuned/ to /etc/tuned/profiles/
 for f in %{_sysconfdir}/tuned/*; do
-  if [ -f "$f/tuned.conf" ] && [ ! -d "%{_sysconfdir}/tuned/profiles/`basename $f`" ]; then
+  if [ -e "$f/tuned.conf" ] && [ ! -e "%{_sysconfdir}/tuned/profiles/`basename $f`" ]; then
```

Comment:
IMHO the current system profile directory is `/usr/lib/tuned/`, not `/usr/lib/tuned/profiles/`.

But I think you should migrate all profiles, because there could be custom profiles overrides in `/etc/tuned/` with the same name as profiles in `/var/lib/tuned/`. It's quite common to e.g. do:
```
# custom throughput-performance profile in /etc/tuned overriding setting of throughput-performance profile in /var/lib/tuned
[main]
# the stock one
include=throughput-performance

# customization
...
```

Reply:
The second check is actually only there to make sure that we don't overwrite existing directories in `/etc/tuned/profiles/` - though they have no reason to exist before the migration, and I could also do that more cleanly using `mv -n`.